docs/contributor_guide.md

```diff
@@ -118,137 +106,14 @@ There's an [open issue](https://github.com/tweenjs/discuss/issues/3) to track wo
-| --------------- | ----------------------------------------------------------------------- | ----------------- |
-| fix             | fixes a bug but does not change the API                                 | Increases PATCH   |
-| style           | formatting changes                                                      |                   |
-| docs            | adding/removing/changing docs                                           |                   |
-| refactor        | rewriting code but not breaking changes, adding features or fixing bugs |                   |
-| test            | changes in tests, e.g. adding a missing one)                            |                   |
-| feat            | adding new features which do not change the API                         | Increases MINOR   |
-| BREAKING CHANGE | changes the API                                                         | Increases MAJOR   |
-
-### How to install and configure `semantic-release`
-
-This is mostly for informational purposes, as `semantic-release` is already configured with Travis and contributors shouldn't need to worry about this, but it is good to document everything.
-
-#### Option 1: using the CLI utility
-
-First install the global cli utility:
-
-```bash
-npm install -g semantic-release-cli
-```
-
-Then in an existing node.js based project (i.e. a `package.json` already exists):
-
-```bash
-semantic-release-cli setup
-```
-
-It will ask you a series of questions to set up `semantic-release` on the project. If all goes well, the next time you push to GitHub, a new release will automatically happen.
-
-You will need to have TravisCI enabled in your account.
-
-#### Option 2: manually
-
-Install the module:
-
-```bash
-npm install --save-dev semantic-release
-```
-
-Edit `package.json` to add the `semantic-release` script:
-
-```javascript
-"scripts": {
-	//...
-	"semantic-release": "semantic-release pre && npm publish && semantic-release post"
-	//...
-},
-```
-
-Create a `.travis.yml` file if it doesn't exist yet (here is [help creating `travis.yml` files](https://docs.travis-ci.com/user/getting-started/)), or [have a look at ours](https://github.com/tweenjs/tween.js/blob/master/.travis.yml).
-
-Add an `after_success` section to `.travis.yml`, in order to run `semantic-release`:
-
-```yaml
-after_success:
-  - npm run semantic-release
-```
-
-Now we need to enable the project in Travis CI, so make sure you have an account there and are logged in.
-
-Enable the project in Travis (if you're the maintainer) or ask the maintainer to enable it, in the [Travis settings page](https://travis-ci.org/profile/).
-
-Click on the cog near to the project name to configure some options.
-
-Scroll down until you see _Environment Variables_.
-
-Add tokens for `GH_TOKEN` and `NPM_TOKEN`. Make sure both variables are hidden: `Display value in build log` should be `Off`.
+Currently the release process is manual.
```

Comment:
Removed the semantic-release/Travis description, as we're not using it currently. Maybe we can add semantic-release back on top of GitHub actions (later PR).